src/main/flight/mixer.c

```diff
@@ -807,10 +807,9 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs)
     if (featureIsEnabled(FEATURE_MOTOR_STOP)
         && ARMING_FLAG(ARMED)
         && !mixerRuntime.feature3dEnabled
-        && !airmodeEnabled
-        && !FLIGHT_MODE(GPS_RESCUE_MODE | ALT_HOLD_MODE)   // disable motor_stop while GPS Rescue / Altitude Hold is active
+        && !airmodeEnabled                               // not with airmode or launch control
+        && !FLIGHT_MODE(GPS_RESCUE_MODE | ALT_HOLD_MODE) // mot in autopilot modes
```

Comment:
might want a separate function like:
isAutoPilotMode() that returns true/false.
When adding future autopilot modes it will be easy to forget places like this... But could be another PR.

Reply:
Good idea, though I'm not sure where best to put it.

Reply:
> Good idea, though I'm not sure where best to put it.

i guess right next to where we define `GPS_RESCUE_MODE`, `ALT_HOLD_MODE` etc.

Reply:
Guess this needs merge https://github.com/betaflight/betaflight/pull/13954 before